corehq/apps/events/tests/test_views.py

```diff
@@ -15,8 +15,19 @@
 from corehq.util.test_utils import flag_enabled
 from corehq.apps.events.models import AttendanceTrackingConfig
 
-from ..models import Event, get_attendee_case_type
-from ..views import EventCreateView, EventsView, AttendeesConfigView
+from ..models import (
+    Event,
+    get_attendee_case_type,
+    AttendeeModel,
+)
+from ..views import (
+    EventCreateView,
+    EventsView,
+    AttendeesConfigView,
+    AttendeeDeleteView
+)
+from ...hqcase.case_helper import CaseHelper
```

Comment:
It's fine, don't change what you've done, but that's not quite what I meant. It's perfectly fine to use CaseHelper or CaseFactory + CaseStructure. I just meant keep relative imports for "this app" and use normal imports for everything else, so that they're sorted logically: Standard library at the top, then third-party libraries, then CommCare stuff, then "this app" stuff. (You see, definitely nitpicky. :grimacing: )

Reply:
Ah, that makes sense. Apologies, I completely misunderstood then! :smile: I do still it is a worthwhile change having removed it as it reduces the amount of dependency on external apps in the project (which is always a good thing for tests).